spec/views/catalog/show.html.erb_spec.rb

```diff
@@ -40,15 +38,13 @@
       visit 'catalog/6109323'
       expect(page.find('#ark_array_id')['data-ark']).to eq('["ark:/88435/0v8382995"]')
       expect(page).to have_selector('#ark_array_id')
-      expect(page).to render_template(partial: '_viewer_uv')
       expect(page).to have_selector('div#view')
     end
 
     it 'will display 2 viewers from two arks' do
```

Comment:
@jrgriffiniii a heads up. Since the new PR will look up for a bibid in figgy, maybe you should consider this test bibid=3943643z because it is not a correct bibid and could not be in figgy.